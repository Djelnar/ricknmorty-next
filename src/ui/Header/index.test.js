/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'


import { HeaderLayout } from './index'


describe('Header', () => {
  it('should render correctly', () => {
    const output = renderer.create(<HeaderLayout>Mock Content</HeaderLayout>).toJSON()

    expect(output).toMatchSnapshot()
  })
})
