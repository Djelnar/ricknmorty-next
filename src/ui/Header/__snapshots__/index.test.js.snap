// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 16px 0;
  background-color: blue;
}

<header
  className="c0"
>
  Mock Content
</header>
`;
